src/v3/src/components/PhoneAuthenticator/PhoneAuthenticator.tsx

```diff
@@ -69,7 +72,7 @@ const PhoneAuthenticator: UISchemaElementComponent<{
           type="text"
           data-se="extension"
           name="extension"
-          label={t('phone.extention.label')}
+          label={loc('phone.extention.label', 'login')}
```

Comment:
loc still in component

Reply:
This is the last component that uses loc, will slack, need some advice on the best approach to handle this.